CHANGELOG

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.2.3] - 2024-03-05
+
+### Features
+
+- Require superuser in pg_idkit.control
```

Comment:
In terms of features, better wording is

"Ability to install the extension for non-root users"

Reply:
Just added this in parens! Thanks for the note